internal/harnesses/container/container.go

```diff
@@ -47,7 +47,8 @@ func (h *container) Destroy(ctx context.Context) error {
 // StepFn implements types.Harness.
 func (h *container) StepFn(config types.StepConfig) types.StepFn {
 	return func(ctx context.Context) (context.Context, error) {
-		log.Info(ctx, "stepping in container", "command", config.Command)
+		log := clog.FromContext(ctx)
+		log.Infof("stepping in container with command %s", config.Command)
```

Comment:
Can you keep this as Info() so it correctly attaches the command? %s is going to have a hard time rendering the arbitrary commands

Reply:
`Info()` won't work because it doesn't plumb the context through